homeassistant/components/zha/cover.py

```diff
@@ -213,35 +213,6 @@ async def async_stop_cover_tilt(self, **kwargs: Any) -> None:
         """Stop the cover tilt."""
         await self.async_stop_cover()
 
-    async def async_update(self) -> None:
```

Comment:
I think this can be removed thanks to `super().async_update()` calls `cluster_handler.XXX.async_update()`, which (in this case) also reads attribute values and updates the state.